stock_move_line_auto_fill/README.rst

```diff
@@ -0,0 +1,143 @@
+=========================
+Stock Move Line Auto Fill
+=========================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fstock--logistics--workflow-lightgray.png?logo=github
+    :target: https://github.com/OCA/stock-logistics-workflow/tree/12.0/stock_move_line_auto_fill
+    :alt: OCA/stock-logistics-workflow
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/stock-logistics-workflow-12-0/stock-logistics-workflow-12-0-stock_move_line_auto_fill
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/154/12.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module alow auto fill quantities in picking operations and autoassignment
+lots quantities.
+
+In Odoo, if you schedule to transfer 50 product quantities and only receive 49
+quantities, you have to change the quantity directly on the picking.
+As the quantity by default is 0 for each line, you have to write the received
+quantity on 49 lines.
+
+In this module we have added a button that helps users to fill automatically
+the scheduled quantities. Then, the user can just change back the quantities
+for the product that hasn't been received yet.
+
+Products with lots
+==================
+When working with lots, it's very uncomfortable to introduce the quantity,
+lot by lot, when transferring pickings from your warehouse (outgoing or
+internal).
+
+This module automatically assigns the reserved quantity as the done one, so
+that you only have to change it in case of divergence, but having the
+possibility of transferring directly.
+
+Also this module adds a button in backorder confirmation wizard to auto
```

Comment:
Does this module actually have the feature specified here, I can't find new button in code? "A button in back order confirmatoin wizard to auto ... "

Reply:
Yes, that's true, I removed that paragraph from the README